catalog/nsili/catalog-nsili-source/src/main/java/org/codice/alliance/nsili/source/NsiliSource.java

```diff
@@ -921,6 +954,15 @@ private boolean isAttributeSupported(String attribute) {
         return false;
     }
 
+    private boolean isIorUrlHttpProtocol() {
+        return (StringUtils.startsWith(iorUrl, "http://") || StringUtils.startsWith(iorUrl,
```

Comment:
Nit - would it be better to check the protocol of a `URL` object or the scheme of a `URI`?